glustercli/cmd/volume-reset.go

```diff
@@ -0,0 +1,65 @@
+package cmd
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/gluster/glusterd2/pkg/api"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+const (
+	volumeResetCmdHelpShort = "Reset volume options"
+	volumeResetCmdHelpLong  = "Reset options on a specified gluster volume. Needs a volume name and at least one option"
+)
+
+var (
+	flagForce, flagResetAll bool
+
+	volumeResetCmd = &cobra.Command{
+		Use:   "reset <volname> <option>",
+		Short: volumeResetCmdHelpShort,
+		Long:  volumeResetCmdHelpLong,
+		Args:  cobra.RangeArgs(1, 2),
+		Run: func(cmd *cobra.Command, args []string) {
+			var req api.VolOptionResetReq
+			volname := args[0]
+			if flagForce {
+				req.Force = true
```

Comment:
This can be simplified:

```go
req := api.VolOptionResetReq {
    Force: flagForce,
    All: flagResetAll,
}
```

Do validation here to prevent specifying an option with `All`

Reply:
@prashanthpai  Still confused about whether to give out an error in case user has specified an option along with ALL or to go for a more subtle way and ignore the option specified.

Reply:
I'm okay with that, if you ignore.